pact_broker/Gemfile

```diff
@@ -1,6 +1,5 @@
 source 'https://rubygems.org'
 
 gem "pact_broker", "~>2.0"
-gem "pg"
 gem "passenger"
-
+gem "mysql2", "0.3.16"
```

Comment:
This should allow patch releases so that any bug fixes are automatically updated. Please change to "~>0.3.16"